sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -9633,6 +10251,112 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRule>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class NSPConfigAccessRuleProperties : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>
+    {
+        public NSPConfigAccessRuleProperties() { }
+        public System.Collections.Generic.IList<string> AddressPrefixes { get { throw null; } }
+        public string Direction { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> FullyQualifiedDomainNames { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.Sql.Models.NSPConfigNetworkSecurityPerimeterRule> NetworkSecurityPerimeters { get { throw null; } }
+        public System.Collections.Generic.IList<string> ServiceTags { get { throw null; } }
+        public System.Collections.Generic.IList<string> Subscriptions { get { throw null; } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAccessRuleProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class NSPConfigAssociation : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Sql.Models.NSPConfigAssociation>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Sql.Models.NSPConfigAssociation>
```

Comment:
`NSP` is short for `NetworkSecurityPerimeter`, consider expanding it to the full name here.
And prepend `Sql` or others to avoid name dup across RP.

Reply:
Renamed to SqlNetworkSecurityPerimeterConfigAssociation